src/main.rs

```diff
@@ -375,7 +375,11 @@ fn main() {
         cmd_start(config, &mut tree, branch_name, &cwd)
       }
 
-      ("upload", Some(submatches)) => unimplemented_subcommand("upload"),
+      ("upload", Some(submatches)) => {
+        let cwd = std::env::current_dir().context("failed to get current working directory")?;
+        let mut tree = Tree::find_from_path(cwd.clone())?;
```

Comment:
Instead of using cwd, add a required path argument and figure out the repository from there, so we can extend it to do something more like repo's interactive thing without breaking workflows? (I always just use `repo upload . --cbr`, so I don't know whether people actually want the interactive thing...)

Reply:
Yeah, I was unsure about that as well. I only ever use the `--cbr .` workflow too. Easy enough to add though.

Do you want me to add the skeleton for the interactive bit as well, or just add the `--cbr` argument and halt with an error if it's not passed?

Reply:
> Do you want me to add the skeleton for the interactive bit as well? That is, add the --cbr argument and halt with an error if it's not passed?

Yeah, that's probably a good idea.

Reply:
Alright, both done. `--cbr` is required, as is a (single) path.

Unfortunately clap doesn't support multiple names for `--` options cleanly, so I didn't add `--current-branch`. If you think that's important it's simple enough to add both options and or them, but maybe it'd be good to remove some duplicate spellings? Lmk